components/tooltip/tooltip.js

```diff
@@ -357,6 +363,92 @@ class Tooltip extends RtlMixin(LitElement) {
 		});
 	}
 
+	async updatePosition() {
```

Comment:
Is this now meant to be more of a "public" method that consumers would call?

Reply:
Its used publicly in one place that I'm hoping to go in and remove so it can go back to being private.